backend/models/invite_user_adm.py

```diff
@@ -63,9 +65,47 @@ def send_notification(self, current_institution):
             notification_type=notification_type
         )
     
-    def send_response_notification(self, current_institution, action):
+    def create_system_notification(self):
+        """Create a new system notification to transfer admin permissions."""
```

Comment:
Create a new system notification for the new administrator to inform you that administrative permissions have been transferred.